sdk/core/Azure.Core/src/Pipeline/MultipartResponse.cs

```diff
@@ -38,8 +39,39 @@ internal static class Multipart
         /// <summary>
         /// Parse a multipart/mixed response body into several responses.
         /// </summary>
-        /// <param name="batchContent">The response content.</param>
-        /// <param name="batchContentType">The response content type.</param>
+        /// <param name="response">The response containing multi-part content.</param>
+        /// <param name="expectCRLF">Controls whether the parser will expect all multi-part boundaries to use CRLF line breaks. This should be true unless more permissive line break parsing is required.</param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate notifications
+        /// that the operation should be cancelled.
+        /// </param>
+        /// <returns>The parsed <see cref="Response"/>s.</returns>
+        public static Response[] Parse(
+            Response response,
+            bool expectCRLF,
```

Comment:
expectWindowsNewLine?

Reply:
My opinion here is that `WindowsNewLine` could be a bit obscure (Is it clear that this means CRLF?). Curious to hear @KrzysztofCwalina 's thoughts.

Reply:
Is this parameter essentially saying whether the boundary will start with a CrLf?

Reply:
yes - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/Multipart/MultipartBoundary.cs#L31-L38

Reply:
The RFC seems to suggest that this is required - https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html. Is there a use case for omitting this?

Reply:
Somebody forgot to tell the service about this requirement :)